pkg/syncer/util.go

```diff
@@ -56,6 +56,31 @@ func getBoundPVs(ctx context.Context, metadataSyncer *metadataSyncInformer) ([]*
 	return boundPVs, nil
 }
 
+// getInlineMigratedVolumesInfo is a helper function for retrieving  inline PV information from Pods
+func getInlineMigratedVolumesInfo(ctx context.Context, metadataSyncer *metadataSyncInformer) (map[string]string, error) {
+	log := logger.GetLogger(ctx)
+	inlineVolumes := make(map[string]string)
+	// Get all Pods from kubernetes
+	allPods, err := metadataSyncer.podLister.List(labels.Everything())
+	if err != nil {
+		return nil, err
```

Comment:
Can you log err message here before returning err?

Reply:
Done